starburst-presto/presto.sh

```diff
@@ -22,8 +22,8 @@ readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
 readonly PRESTO_MASTER_FQDN="$(/usr/share/google/get_metadata_value attributes/dataproc-master)"
 readonly WORKER_COUNT=$(/usr/share/google/get_metadata_value attributes/dataproc-worker-count)
 readonly PRESTO_BASE_URL=https://storage.googleapis.com/starburstdata/presto
-readonly PRESTO_MAJOR_VERSION="312"
-readonly PRESTO_VERSION="${PRESTO_MAJOR_VERSION}-e.1"
+readonly PRESTO_MAJOR_VERSION="323"
```

Comment:
```
wget https://storage.googleapis.com/starburstdata/presto/323e/323-e.3/presto-server-323-e.3.tar.gz
2020-01-11 17:49:50 ERROR 403: Forbidden.
```
Hey, we talked before right; uploading `presto-server-323-e.3.tar.gz` to GCS is not in the pipeline of the Starburstdata's release cycle...

Reply:
Should be accessible now.

Reply:
Awesome :)

Reply:
@ChethanUK thanks for checking me!